scripts/check_test_datasets.py

```diff
@@ -0,0 +1,49 @@
+#! /usr/bin/env python3
+import subprocess
+import sys
+from dataclasses import dataclass
+from pathlib import Path
+
```

Comment:
nit: I'm a big fan of the convention of writing a block comment at the top of these things stating what its use is, for the uninitiated it can be unclear what this is checking.